pipes/signal_minute.pipe

```diff
@@ -8,17 +8,14 @@ SQL >
     SELECT 
         toStartOfMinute(time) AS time,
         network,
-        asset,
+        if (signal = '', asset, signal) AS asset,
         metric,
         value,
         unit,
         job_id,
         ingested_at
     FROM signals
+    WHERE asset != ''
```

Comment:
This is to filter invalid data: e.g: Sending asset empty and no signal field. It's ok to filter the wrong data but we don't have a mechanism to warn the user.